### DISCUSSION TITLE
Edit AI responses instead of regenerating them

### DISCUSSION THREAD

**Discussion by LeonelRFF** [logancyang/obsidian-copilot#1169] [Q&A]
**Is your feature request related to a problem? Please describe.**
It is not possible to edit the AI's answers

**Describe the solution you'd like**
is it possible to edit the answers that the AI ​​returns to me before saving them?
Use case, on one occasion the AI ​​created a diagram with a single error, I just had to add "`>`" and that's it, maybe add a button that allows this, it's more practical without having to generate the entire answer again.

**Describe alternatives you've considered**
Related topic to this, if this can be added, then customizable buttons could be added that users program, (unless what I say can be done are templates) Let me explain better, copilot-obsidian helps me understand a difficult topic, with the customizable button I would send only a specific message to the [quiz generator](https://github.com/ECuiDev/obsidian-quiz-generator) plugin, which also uses AI, to transform the message into a review card, useful for students 😊

**Additional context**
The [Page assistan](https://github.com/n4ze3m/page-assist) chrome extension allows you to edit the responses

**Comment by logancyang** [comment]
This is not a priority at the moment but you can always save the conversation, edit in markdown and reload it if needed.
